src/theme/components/index.tsx

```diff
@@ -467,17 +467,19 @@ export const SpinnerSVG = styled.svg`
   ${SpinnerCss}
 `
 
-const BackArrowLink = styled(StyledInternalLink)`
-  color: ${({ theme }) => theme.textPrimary};
-`
-export function BackArrow({ to }: { to: string }) {
+export function BackArrowLink({ to }: { to: string }) {
   return (
-    <BackArrowLink to={to}>
-      <ArrowLeft />
-    </BackArrowLink>
+    <StyledInternalLink to={to}>
+      <BackArrowIcon />
+    </StyledInternalLink>
   )
 }
 
+export const BackArrowIcon = styled(ArrowLeft)`
+  color: ${({ theme }) => theme.textPrimary};
+  cursor: pointer;
```

Comment:
I don't think `cursor: pointer` should be set on non-clickable elements by default. You can safely remove it, because `StyledInternalLink` that you wrap `BackArrowIcon` with has `cursor: pointer` already set up and that value will be propagated to the children too.